scripts/redshift/load_redshift.py

```diff
@@ -0,0 +1,44 @@
+import os
+
+from matrix.common.etl import run_etl, transform_bundle, finalizer_reload
+
+
+if __name__ == '__main__':
+    # All SS2 analysis bundles
+    query = {
+        "query": {
+            "bool": {
+                "must": [
+                    {
+                        "match": {
+                            "files.library_preparation_protocol_json.library_construction_approach.ontology": "EFO:0008931"
```

Comment:
I'm guessing this will need to change with the recent metadata change?

Reply:
Yes